scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+OP="$1"
+rule="RH-Firewall-1-INPUT -p tcp -m conntrack --ctstate NEW -m tcp --dport 6640 -j ACCEPT"
+
+case "${OP}" in
+    start)
+        if iptables -S | grep -e "${rule}" > /dev/null
+        then
+            echo "port 6640 is opened already"
+            exit 1
+        else
+            iptables -A ${rule}
```

Comment:
@taoyongd After running that command, and checking with `iptables -L`, the chain looks like this:

```
Chain RH-Firewall-1-INPUT (2 references)
target     prot opt source               destination         
ACCEPT     all  --  anywhere             anywhere            
ACCEPT     icmp --  anywhere             anywhere             icmp any
ACCEPT     udp  --  anywhere             anywhere             udp dpt:bootps
ACCEPT     all  --  anywhere             anywhere             ctstate RELATED,ESTABLISHED
ACCEPT     udp  --  anywhere             anywhere             ctstate NEW udp dpt:ha-cluster
ACCEPT     tcp  --  anywhere             anywhere             ctstate NEW tcp dpt:ssh
ACCEPT     tcp  --  anywhere             anywhere             ctstate NEW tcp dpt:http
ACCEPT     tcp  --  anywhere             anywhere             ctstate NEW tcp dpt:https
ACCEPT     tcp  --  anywhere             anywhere             tcp dpt:21064
ACCEPT     udp  --  anywhere             anywhere             multiport dports hpoms-dps-lstn,netsupport
REJECT     all  --  anywhere             anywhere             reject-with icmp-host-prohibited
ACCEPT     tcp  --  anywhere             anywhere             ctstate NEW tcp dpt:6640
```
So your new rules appear after `REJECT all`, as @anoobs predicted. I don't see how that can work.